app/components/request_list.tsx

```diff
@@ -22,7 +22,7 @@ const RequestList = (props: RequestObj) => {
   const columns = [
     {
       Header: "Request URL",
-      accessor: "url",
+      Cell: ({original}) => <div style={{flex: '100 0 auto', width: '100px'}}className="url" title={original.url}>{original.url}</div>,
```

Comment:
possible to move the style to class `url`?

Reply:
Yeah